learn/app-development/variables/custom-formatter.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Custom Formatter"
+id: ""
+sidebar_label: "Custom Formatter"
+---
+With Custom Formatter, customize data more specific to your audience.
+
+---
+
+Convert data received from services to a well-formatted data before showing it in the UI, depending on the role, country, currency, and more. For example, John is a card-holder, he can view his card details without any restrictions, whereas the bank admin cannot see his card number which is masked with specific character  as shown below.
+
```

Comment:
can we also mention, WaveMaker provides a few default formatters (visible in the "Use Expression" tab). Sometimes when these do not meet the requirement, users can write their own custom formatters.

Reply:
@vibhus added the requested change.

![image](https://user-images.githubusercontent.com/25283489/85837448-a4332400-b7b5-11ea-9ebd-34613803f563.png)